docs/getting-started/launching-one-node/crypto-materials.md

```diff
@@ -9,3 +9,66 @@ title: Creating Crypto Material
  SPDX-License-Identifier: CC-BY-4.0
  -->
 
+### Crypto materials configuration
+
+Hyperledger Orion uses public key cryptography from user side, for user authentication and transaction Non-Repudiation. From server side, Orion uses PKI to sign and verify all server responses to clients. 
+
+Thus, each Orion client needs certificate/key pair to sign each query and transaction he submits to server, and server has to have its own certificate/key pair to sign each response to client.
+
+From another side, each Orion server has to keep each client certificate to validate client signatures and each client has to have access to server certificate to validate servers signatures. Both server and client need access to CA certificate to validate new client/server certificates.
```

Comment:
In order to validate client signatures, the Orion server has to keep the certificate of every client. This is achieved by the process of user on-boarding (add link to user management). Conversely, in order to validate the server's signature on responses, each client has to have access to server certificate. This is achieved by reading the ClusterConfig.Nodes (link to API). Both server and client need access to a CA certificate chain to validate client/server (resp.) certificates.